portal-bridge/src/beacon_bridge.rs

```diff
@@ -138,11 +165,56 @@ impl BeaconBridge {
         });
     }
 
-    #[allow(dead_code)] // FIXME: Remove this once we use it
-    async fn serve_light_client_bootstrap(&self, block_root: String) -> anyhow::Result<()> {
-        let data = self.api.get_lc_bootstrap(block_root).await?;
-        let update: Value = serde_json::from_str(&data)?;
-        let _: LightClientBootstrapCapella = serde_json::from_value(update["data"].clone())?;
+    /// Serve `LightClientBootstrap` data
+    async fn serve_light_client_bootstrap(
+        api: ConsensusApi,
+        portal_clients: Vec<HttpClient>,
+        finalized_block_root: Arc<Mutex<String>>,
+    ) -> anyhow::Result<()> {
+        let response = api.get_beacon_block_root("finalized".to_owned()).await?;
+        let response: Value = serde_json::from_str(&response)?;
+        let latest_finalized_block_root: String =
+            serde_json::from_value(response["data"]["root"].clone())?;
+
+        // If the latest finalized block root has changed, serve a new bootstrap
+        if !finalized_block_root
+            .lock()
+            .expect("Failed to lock finalized block root")
+            .eq(&latest_finalized_block_root)
+        {
+            let result = api.get_lc_bootstrap(&latest_finalized_block_root).await?;
+            let result: Value = serde_json::from_str(&result)?;
+            let bootstrap: LightClientBootstrapCapella =
+                serde_json::from_value(result["data"].clone())?;
+
+            info!(
+                "Got lc bootstrap for slot {:?}",
+                bootstrap.header.beacon.slot
+            );
+
+            let mut content_bytes = ForkName::Capella.as_fork_digest().to_vec();
+            content_bytes.append(&mut bootstrap.as_ssz_bytes());
+
+            let content_value = BeaconContentValue::decode(&content_bytes)?;
```

Comment:
Something feels backwards about building the serialized value here just to throw it away for the decoded version. It seems more direct to add a new() so we can do something like:
```suggestion
            let content_value = BeaconContentValue::LightClientBootstrap(ForkVersionedLightClientBootstrap::new(bootstrap));
```
Since the bootstrap is typed with the fork, it seems like we can match on it inside `new` to determine the fork.

As a side-benefit, this code block becomes agnostic to the ssz encoding, which seems like what we want.

Reply:
Adding `From<LightClientBootstrapCapella> for ForkVersionedLightClientBootstrap` did the trick, we can then use `.into()` instead of `new()`.